assets/styles/components/_quote-and-image.scss

```diff
@@ -0,0 +1,35 @@
+.circle_flourish {
+    background: var(--sub-color);
+    border-radius: 50%;
+    content: "";

```

Comment:
This is unnecessary as this isn't a pseudoelement (`content: ""` is only necessary for pseudoelements)